package.json

```diff
@@ -139,6 +144,10 @@
         {
           "command": "hexEditor.goToOffset",
           "when": "hexEditor:isActive"
+        },
+        {
+          "command": "hexEditor.selectEditMode",
+          "when": "hexEditor:isActive"
         }
       ],
       "editor/title": [
```

Comment:
We could also register the "Insert" key as the traditional default key binding to change the edit mode (`"when": "hexEditor:isActive"`)

Reply:
How did I completely forget about the insert key. 😅